updates Child Workflow Execution descriptions and samples (#1629)
temporalio/documentation: docs-src/typescript/how-to-spawn-a-child-workflow-execution-in-typescript.md

---
id: how-to-spawn-a-child-workflow-execution-in-typescript
title: How to spawn a Child Workflow Execution in Typescript
sidebar_label: Child Workflow Execution
tags:
  - developer-guide
  - typescript
---

> To start a Child Workflow and return a client-side handle that implements a Child Workflow interface, use [`startChild`](https://typescript.temporal.io/api/namespaces/workflow/#startchild).

Reviewer: ```suggestion
To start a Child Workflow and return a [handle](https://typescript.temporal.io/api/interfaces/workflow.childworkflowhandle/) to it, use [`startChild`](https://typescript.temporal.io/api/namespaces/workflow/#startchild).
```

To start a Child Workflow Execution and await its completion, use [`executeChild`](https://typescript.temporal.io/api/namespaces/workflow/#executechild).

> By default, a child will be scheduled on the same Task Queue as the parent.

Reviewer 2: ```suggestion
By default, a child is scheduled on the same Task Queue as the parent.
```
